bedrock/settings/__init__.py

```diff
@@ -45,5 +45,16 @@
         'CULL_FREQUENCY': 4,  # 1/4 entries deleted if max reached
     }
 }
+# cache for static files
+CACHES['staticfiles'] = {
```

Comment:
Just doing this until we can move to Django 1.7 or add Whitenoise and use the manifest cache.